src/test/resources/application.conf

```diff
@@ -1,6 +1,8 @@
 driver="org.postgresql.Driver"
-username="tdr"
-password="password"
-url="jdbc:postgresql://localhost:5432/consignmentapi"
+username="dGRy" # tdr
+password="cGFzc3dvcmQ=" # password
+url="amRiYzpwb3N0Z3Jlc3FsOi8vbG9jYWxob3N0OjU0MzIvY29uc2lnbm1lbnRhcGk=" # jdbc:postgresql://localhost:5432/consignmentapi
 consignment-api-user = "consignment_api_user"
```

Comment:
Do all the values need to be encrypted? If not maybe state this in the README

Reply:
Only ones that come from environment variables in main application.conf. I've put a line in the README to say this.